src/main/kotlin/org/uqbar/tareas/domain/Tarea.kt

```diff
@@ -45,10 +42,12 @@ class Tarea : Entity() {
     }
 
     @JsonProperty("fecha")
-    fun getFechaAsString() = formatter.format(this.fecha)
+    fun getFechaAsString(): String = formatter.format(this.fecha)
```

Comment:
no termino de entender si tira un warning o por algo en especial le ponés el tipo

Reply:
> no termino de entender si tira un warning o por algo en especial le ponés el tipo

Si, me había aparecido un warning de Kotlin por no tiparlo explicitamente. Como `this.format`/`DateTimeFormatter ` es un método de Java y no kotlin, entonces implicitamente considera lo que retorna un "platform type" `String!` (que puede o no ser nullable, no puede deducirlo automaticamente - formatter.format(...) solo toma strings no nulos y devuelve un string no nulo, pero todo eso es información que kotlin no puede obtener al parecer)